types.go

```diff
@@ -69,6 +69,8 @@ type Config struct {
 	// Theoretically this would go away if/when we standardize on OCI.
 	UnpackRootfs bool `json:"unpack_rootfs" envconfig:"optional"`
 
+	OutputOCI bool `json:"output_oci" envconfig:"optional"`
```

Comment:
@jmccann here you go!

@rdrgmnzs please update readme for this configuration. Thx!